<ui version="4.0" >
 <class>OmgDataFetcherWizardSearchWidgetBase</class>
 <widget class="QWidget" name="OmgDataFetcherWizardSearchWidgetBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="lblSource" >
     <property name="text" >
      <string>Source</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2" >
    <widget class="QComboBox" name="cboScraperPluginName" />
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="lblSearchPhrase_2" >
     <property name="text" >
      <string>Search term</string>
     </property>
     <property name="buddy" >
      <cstring>leSearchString</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QLineEdit" name="leSearchString" >
     <property name="font" >
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="2" >
    <widget class="QToolButton" name="toolAddSearchItem" >
     <property name="text" >
      <string>...</string>
     </property>
     <property name="icon" >
      <iconset resource="../resources/resources.qrc" >:/edit_add.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="lblSearchList_2" >
     <property name="text" >
      <string>Search list</string>
     </property>
    </widget>
   </item>
   <item rowspan="3" row="2" column="1" >
    <widget class="QListWidget" name="lstSearch" >
     <property name="font" >
      <font>
       <family>Sans Serif</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="selectionMode" >
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="2" >
    <widget class="QToolButton" name="toolSelectSearchFileName" >
     <property name="text" >
      <string/>
     </property>
     <property name="icon" >
      <iconset resource="../resources/resources.qrc" >:/fileopen.png</iconset>
     </property>
    </widget>
   </item>
   <item rowspan="2" row="3" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>138</width>
       <height>185</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="2" >
    <widget class="QToolButton" name="toolDeleteItem" >
     <property name="text" >
      <string/>
     </property>
     <property name="icon" >
      <iconset resource="../resources/resources.qrc" >:/edit_remove.png</iconset>
     </property>
    </widget>
   </item>
   <item row="4" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>30</width>
       <height>149</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0" >
    <widget class="QLabel" name="lblOutputFilename_2" >
     <property name="text" >
      <string>Output path</string>
     </property>
     <property name="buddy" >
      <cstring>leOutputPath</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1" >
    <widget class="QLineEdit" name="leOutputPath" >
     <property name="font" >
      <font>
       <family>Sans Serif</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
    </widget>
   </item>
   <item row="5" column="2" >
    <widget class="QToolButton" name="toolSelectOutputPath" >
     <property name="text" >
      <string/>
     </property>
     <property name="icon" >
      <iconset resource="../resources/resources.qrc" >:/fileopen.png</iconset>
     </property>
    </widget>
   </item>
   <item row="6" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Require a minimum of</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="2" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QSpinBox" name="spinMinimumRecords" >
       <property name="toolTip" >
        <string>Search results not meeting this record count requirement will be ignored</string>
       </property>
       <property name="minimum" >
        <number>1</number>
       </property>
       <property name="maximum" >
        <number>99999</number>
       </property>
       <property name="value" >
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>records per taxon</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resources.qrc" />
 </resources>
 <connections/>
</ui>
